api/AudioBufferSourceNode.json

```diff
@@ -378,7 +378,7 @@
               "version_added": null
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
I wasn't able to find the version data for versions 1-3 since they were before my time. Although I just discovered it in ./browsers so I will update all instances accordingly.